modules/navier_stokes/src/kernels/INSChorinPredictor.C

```diff
@@ -91,7 +91,7 @@ INSChorinPredictor::INSChorinPredictor(const InputParameters & parameters)
     // Required parameters
     _component(getParam<unsigned>("component")),
     _predictor_type(getParam<std::string>("predictor_type")),
-    _predictor_enum("OLD, NEW, STAR, INVALID", _predictor_type),
+    _predictor_enum("OLD NEW STAR INVALID", _predictor_type),
```

Comment:
This should be fixed. You are reading in a string and then populating an enum. I'd recommend that you setup the enum in the validParams (gives the user a picklist and validates entry), and then get the enum back out like this:
https://github.com/idaholab/moose/blob/devel/framework/src/postprocessors/NumDOFs.C#L32

Reply:
Minor: I'd also recommend setting up an enum for the component too, but I'll leave that up to you. We do that in GeneratedMesh. You can also just do a rangedChecked param if you like the integer representations.